Refining wording in IAM v2 docs (#2650)
chef/automate: components/automate-chef-io/content/docs/iam-v2-guide.md

#### Configuring Project Limit

> By default, Chef Automate limits to six projects. Use the Chef Automate CLI to increase this project limit.

Reviewer: That seems to be missing an object; how about "limits you"...?
(Same comment applies in the iam-v2-overview.md file below.)

Reviewer 2: ```suggestion
By default, Chef Automate limits you to six projects. You can increase the project limit using the CLI: 
```
